Adds missing @abs_top_srcdir@ to test-cmd-info-variables.sh.in
--- a/test/unit/test-cmd-info-variables.sh.in
+++ b/test/unit/test-cmd-info-variables.sh.in
@@ -32,11 +32,11 @@
     # FIXME try -x -i, and no options. try invalid opts
 }
 
-if [ '/src/external-vcs/sourceforge/bashdb' = '' ] ; then
+if [ '@abs_top_srcdir@' = '' ] ; then
   echo "Something is wrong: abs_top_srcdir is not set."
  exit 1
 fi
-abs_top_srcdir=/src/external-vcs/sourceforge/bashdb
+abs_top_srcdir=@abs_top_srcdir@
 # Make sure $abs_top_src has a trailing slash
 abs_top_srcdir=${abs_top_srcdir%%/}/
 . ${abs_top_srcdir}test/unit/helper.sh
